src/reporter/errors/templates.js

```diff
@@ -261,6 +261,30 @@ export default {
         ${getScreenshotInfoStr(err)}
     `,
 
+    [TYPE.incorrectGlobalWaitForActionEventArgument]: err => dedent`
+        ${getMsgPrefix(err, CATEGORY.actionError)}Error at step <span data-type="step-name">${err.stepName}</span>:
+
+        <code data-type="api">__waitFor</code> action's first parameter should be a function.
+
+        ${getScreenshotInfoStr(err)}
+    `,
+
+    [TYPE.incorrectGlobalWaitForActionTimeoutArgument]: err => dedent`
+        ${getMsgPrefix(err, CATEGORY.actionError)}Error at step <span data-type="step-name">${err.stepName}</span>:
+
+        <code data-type="api">__waitFor</code> action's "timeout" parameter should be a positive number.
+
+        ${getScreenshotInfoStr(err)}
+    `,
+
+    [TYPE.globalWaitForActionTimeoutExceeded]: err => dedent`
+        ${getMsgPrefix(err, CATEGORY.timeout)}Error at step <span data-type="step-name">${err.stepName}</span>:
+
+        <code data-type="api">__waitFor</code> user code timeout exceeded.
```

Comment:
@VasilyStrelyaev Maybe we should use [the same description](https://github.com/DevExpress/testcafe/pull/267/files#diff-4334bbbf0649f8c98cd2becdc41b7599L259) as for the `waitFor` action? They do the same and have the same arguments. The only difference - they are used in different places: `waitFor` in steps only via `act.waitFor`, but `__waitFor` in any user's script (test, test helpers or mixins) as a global variable